redbot/core/bot.py

```diff
@@ -1374,14 +1374,15 @@ def add_command(self, command: commands.Command) -> None:
                 if permissions_not_loaded:
                     subcommand.requires.ready_event.set()
 
-    def remove_command(self, name: str) -> None:
+    def remove_command(self, name: str):
         command = super().remove_command(name)
         if not command:
             return
```

Comment:
```suggestion
        if command is None:
            return command
```
This checks the results of the `super` call more correctly and causes the return types to be exactly that of the `super`